src/styles/utils/variables.scss

```diff
@@ -0,0 +1,16 @@
+/* start the variables of repeated colors */
+
+$layout-color: #00acdc;
+$star-proportion: 16px;
+$color-linkHoverActive: #00acdc;
```

Comment:
There is no need to use a lot of variables for the same color

Reply:
changed